src/gui.cpp

```diff
@@ -0,0 +1,36 @@
+#include "gui.h"
+
+void GUI::clear_screen() {
+  // Mac/Linux
+#ifndef DEBUG
+  system("clear");
+#endif
+}
+
+void GUI::print_border() {
```

Comment:
I'm not going to go too deep into c++ specific things but if you would like:
`std::array`
`std::string`
`std::stringstream`

Reply:
Sure thing, I can rewrite the logic more CPP-esqe if that's more appropriate.